src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import 'vars';
+
+.page__body {
   margin: 0;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  box-sizing: border-box;
+}
```

Comment:
The `.page__body` class seems to be a general class for the body of your page. It would be better to move this to a separate file (e.g., `_general.scss`) to keep your code modular and organized.